### DISCUSSION TITLE
Is it required to archive and retain old MRFs from previous months?

### DISCUSSION THREAD

**Discussion by OHCDiana** [CMSgov/price-transparency-guide#541] [Q&A] [answered]
In a scenario involving a network, a TPA, and an employer group, are any of these entities required to archive and retain old versions of MRFs from previous months? If so, which of those 3 entities and for how long?

**Comment by Devyandu** [comment, marked as answer]
Here is a previous thread on this topic with some responses. 
[https://github.com/CMSgov/price-transparency-guide/discussions/238](https://github.com/CMSgov/price-transparency-guide/discussions/238)
